azure/route.sh

```diff
@@ -12,6 +12,36 @@ fn azure_route_table_delete(name, group) {
 	)
 }
 
+fn azure_route_table_route_new(name, group, routetable, address, hoptype) {
+	instance = (
+		"--name"
+		$name
+		"--resource-group"
+		$group
+		"--route-table-name"
+		$routetable
+	    "--address-prefix"
+		$address
+	    "--next-hop-type"
+		$hoptype
+	)
+
+	return $instance
+}
+
+fn azure_route_table_route_set_hop_address(instance, hopaddress) {
+	instance <= append($instance, "--next-hop-ip-address")
+	instance <= append($instance, $hopaddress)
+
+	return $instance
+}
+
+fn azure_route_table_route_create(instance) {
+	azure network route-table route create $instance
+}
+
+# azure_route_table_add_route Create route in a Route Table
+# deprecated: Use the azure_route_table_route_new function
```

Comment:
@ppizarro @katcipis Sorry for looking at this just now. But I was thinking why does this need to be deprecated? Maybe we can do something about the function naming, but I think that the use of `--next-hop-ip-address` is pretty standard for most use cases.